File: Clinica/Persona.cs
/*
 * Created by SharpDevelop.
 * User: accia
 * Date: 26/5/2022
 * Time: 12:56
 * 
 * To change this template use Tools | Options | Coding | Edit Standard Headers.
 */
using System;

namespace Clinica
{
	/// <summary>
	/// Description of Persona.
	/// </summary>
	public class Persona
	{
		private string Nombre, Apellido;
		private int Dni;
		
		public Persona(string Nombre,string Apellido,int Dni)
		{
			this.Nombre = Nombre;
			this.Apellido = Apellido;
			this.Dni = Dni;
		}
		
		
		public string nombre{
			get{
				return Nombre;
			}
		}
		
		public string apellido{
			get{
				return Apellido;
			}
		}
		
		public int dni{
			get{
				return Dni;
			}
		}
	}
}

File: Clinica.ConvertedToPython/Persona.py
# 
# * Created by SharpDevelop.
# * User: accia
# * Date: 26/5/2022
# * Time: 12:56
# *
# * To change this template use Tools | Options | Coding | Edit Standard Headers.
# 
from System import *

class Persona(object):
	""" <summary>
	 Description of Persona.
	 </summary>
	"""
	def __init__(self, Nombre, Apellido, Dni):
		self._Nombre = Nombre
		self._Apellido = Apellido
		self._Dni = Dni

	def get_nombre(self):
		return Nombre

	nombre = property(fget=get_nombre)

	def get_apellido(self):
		return Apellido

	apellido = property(fget=get_apellido)

	def get_dni(self):
		return Dni

	dni = property(fget=get_dni)